service-commands/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Library for performing commands against services, including setup.",
   "main": "src/index.js",
   "scripts": {
-    "preinstall": "mkdir -p ~/.local/bin && ln -f scripts/A ~/.local/bin/A"
+    "preinstall": "mkdir -p ~/.local/bin && ln -sf $PWD/scripts/A ~/.local/bin/A"
```

Comment:
Just to confirm since this is symlinked if the script in the scripts folder if you pull the cli will automatically call the new changes right? Cause you're not moving the files to bin?

Reply:
Yes, that is correct